src/bun.js/bindings/JSBufferList.cpp

```diff
@@ -183,6 +183,17 @@ JSC::GCClient::IsoSubspace* JSBufferList::subspaceForImpl(JSC::VM& vm)
 
 STATIC_ASSERT_ISO_SUBSPACE_SHARABLE(JSBufferListPrototype, JSBufferListPrototype::Base);
 
+template<typename Visitor>
+void JSBufferList::visitChildrenImpl(JSCell* cell, Visitor& visitor)
+{
+    JSBufferList* buffer = jsCast<JSBufferList*>(cell);
+    ASSERT_GC_OBJECT_INHERITS(buffer, info());
+    Base::visitChildren(buffer, visitor);
+    for (auto& val : buffer->m_deque)
```

Comment:
This won't cause it to clear the queue right?

Reply:
> This won't cause it to clear the queue right?

it won't